lib/utils/httpDate.js

```diff
@@ -11,9 +11,12 @@
  * @returns  {String}     formatted dated object
  */
 module.exports = function httpDate(now) {
```

Comment:
Yikes, didn't even thinking about that. httpDate isn't exposed to the public and is internal only - doesn't look like the falsy case here is being exercised so I think I will just remove it.